### PR TITLE
Chore: add banner for KubeCon NA 2022

### LINKED ISSUE #36106: Add banner for kubecon NA 2022
**This is a Feature Request**

**What would you like to be added**
An announcement banner for KubeCon NA 2022

**Why is this needed**
KubeCon NA 2022 is around the corner. It is important we start informing the community about this amazing event by displaying a banner on the Kubernetes website.
**Comments**

This change will need steering committee approval.

I would set the banner to start on the 22nd of August and end on the 28th of October, 2022 at 18:00 CEST

/priority important-soon

### PR DESCRIPTION
**Pull request by thisisobate**
Screenshot 2022-08-19 at 13 19 13

fixes #36106

### CODE REVIEW

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -116,3 +116,16 @@ announcements:
   message: |
     5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
     May 16 - 20, 2022.
+
+- name: Kubecon 2022 NA
+  startTime: 2022-08-22T00:00:00 #Added in https://github.com/kubernetes/website/pull/36107
```
Comment (sftim) [inline data/announcements/scheduled.yaml:121]: This seems too early. We usually only turn the banner on around 1 week before the conference starts.

Comment (thisisobate) [reply data/announcements/scheduled.yaml:121]: oh...I see.
Would extend the start date to 17th of October, 2022

File: data/announcements/scheduled.yaml [hunk 2]
```diff
@@ -116,3 +116,16 @@ announcements:
   message: |
     5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
     May 16 - 20, 2022.
+
+- name: Kubecon 2022 NA
+  startTime: 2022-10-15T00:00:00 #Added in https://github.com/kubernetes/website/pull/36107
```
Comment (Sea-n) [inline data/announcements/scheduled.yaml:121]: Not sure why add this unnecessary comment. We can check git log for it.
```suggestion
  startTime: 2022-10-15T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107
```

File: data/announcements/scheduled.yaml [hunk 3]
```diff
@@ -116,3 +116,16 @@ announcements:
...
+  endTime: 2022-10-28T18:00:00
+  style: >-
+    background: linear-gradient(135deg,rgb(2,3,129) 0%,rgb(114,114,255) 100%);
```
Comment (Sea-n) [inline data/announcements/scheduled.yaml:124]: Where did this specific color come from?
```suggestion
    background: linear-gradient(135deg, rgb(2, 3, 129) 0%, rgb(114, 114, 255) 100%);
```

File: data/announcements/scheduled.yaml [hunk 4]
```diff
@@ -116,3 +116,16 @@ announcements:
...
+    color: #fff;
+  title: |
+    <img src ="/images/announcements/kccnc-na-2022-white.svg" style="float: right; height: 80px;" />
+    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon NA 2022</a> <em> Detroit, Michigan + Virtual</em>.
```
Comment (Sea-n) [inline data/announcements/scheduled.yaml:128]:
```suggestion
    <img src="/images/announcements/kccnc-na-2022-white.svg" style="float: right; height: 80px;" />
    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon NA 2022</a> <em>Detroit, Michigan + Virtual</em>.
```

File: data/announcements/scheduled.yaml (hunk 2 above)
Comment (thisisobate) [reply data/announcements/scheduled.yaml:121]: True but I imagined this to be a convention contributors follow when adding a new banner in Kubernetes.io
@sftim any thoughts?

Comment (sftim) [reply data/announcements/scheduled.yaml:121]: The steering committee seem to like it (and they are the approvers for these changes).

```suggestion
  startTime: 2022-10-15T00:00:00 #Added in https://github.com/kubernetes/website/pull/36107 
```

File: data/announcements/scheduled.yaml (hunk 3 above)
Comment (thisisobate) [reply data/announcements/scheduled.yaml:124]: [KubeCon NA 2022 website](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)

File: data/announcements/scheduled.yaml [hunk 5]
```diff
@@ -116,3 +116,16 @@ announcements:
   message: |
     5 days of incredible opportunites to collaborate, learn + share with the entire community!<br />
     May 16 - 20, 2022.
+
+- name: Kubecon 2022 NA
+  startTime: 2022-10-15T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107 
+  endTime: 2022-10-28T18:00:00
```
Comment (reylejano) [inline data/announcements/scheduled.yaml:122]: I suggest to start the Monday of the week before KubeCon and to end at midnight the Saturday after KubeCon

```suggestion
  startTime: 2022-10-17T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107 
  endTime: 2022-10-29T00:00:00
```

Comment (thisisobate) [reply data/announcements/scheduled.yaml:122]: Right...sounds good to me

**Comment by mrbobbytables** [comment]
/approve
/hold
for other steering to weigh in

**Pull request merged**